eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -525,6 +530,8 @@ object Peer {
   case class ConnectionDown(peerConnection: ActorRef) extends RemoteTypes
 
   case class RelayOnionMessage(messageId: ByteVector32, msg: OnionMessage, replyTo_opt: Option[typed.ActorRef[Status]])
+
+  case class ChannelCreationData(localParams: LocalParams)
```

Comment:
This is currently unused?

Reply:
Removed in [4ee1025](https://github.com/ACINQ/eclair/pull/2316/commits/4ee1025748333067f709cbc7c050dc36c5ef169c).